muffnn/autoencoder/autoencoder.py

```diff
@@ -516,13 +516,22 @@ def _build_sigmoid_and_softmax_indices(self):
             raise ValueError("Sigmoid indices and softmax indices cannot"
                              " overlap!")
 
-    def partial_fit(self, X, y=None):
+    def partial_fit(self, X, y=None, monitor=None, **kwargs):
```

Comment:
Why do you pass kwargs here? Is there anything in partial_fit that takes that as an input?

Reply:
Thus is part of the sklearn API. See here http://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn.base.TransformerMixin.fit_transform